src/pages/blog/october-2023/index.mdx

```diff
@@ -0,0 +1,81 @@
+import { ArticleLayout } from '@/components/blog/ArticleLayout'
+
+export const meta = {
+  title: 'Mintlify Product Update October 2023',
+  description:
+    'What we launched in October: Persisting Auth Tokens, CLI Errors Handling, and more. ',
+  mobileImageUrl: '/static/images/blog/august-2023/mobile.png',
+  desktopImageUrl: '/static/images/blog/august-2023/mobile.png',
+  author: 'Josh Kim',
+  authorRole: 'Operations',
+  date: 'October 31, 2023',
+  categories: ['Featured', 'Product'],
+  authorImageUrl: '/static/images/blog/authors/josh.jpg',
+  readingTime: '2 minute read',
+}
+
+export default (props) => <ArticleLayout meta={meta} {...props} />
+
+<img
+  src="/static/images/blog/october-2023/desktop.png"
+  alt="blog thumbnail"
+  className="w-full rounded-md"
+/>
```

Comment:
Not an actionable atm, but I think we need to get a new template for the designs. Not a big fan